run/O2HitMerger.h

```diff
@@ -664,7 +665,7 @@ void O2HitMerger::initDetInstances()
     auto active = std::find(modulelist.begin(), modulelist.end(), s) != modulelist.end();
     return active; };
 
-  mDetectorInstances.resize(DetID::Last);
+  mDetectorInstances.resize(DetID::nDetectors);
```

Comment:
why this change?

Reply:
This I am not 100% sure, but when we add the ITS3 (with DetID=16) the total number of detectors becomes 17. 
The next loop index `i` goes from `DetID::First` (0) to `DetID::Last` (16) included and accesses the `mDetectorInstances[i]`.
If size of `mDetectorInstances` were `DetID::Last` (16) it means that `mDetectorInstances[16]` would not be something legal.
But maybe I missed something but if I remember correctly it did not work until I changed this.

Reply:
I guess you need to adjust DetID::Last when you add some detector. The logic should not be changed here.

Reply:
Ok. Here the change seems ok. I just checked.